resources/js/lang/en.json

```diff
@@ -35,7 +35,8 @@
   "book_status": "Book availability",
   "condition": "Condition",
   "settings_yourInfo": "Your preferred meeting point",
-  "settings_beSafe": "Always try to be safe when meeting people online.",
+  "settings_beSafe": "Always be safe when meeting people online.",
+  "location_sharing": "Booksbank will only ever reveal your preferred location with another person once you have approved a request to loan them your book.",
```

Comment:
the first part of the name is the actual component name that in this case is settings, so you message could be:
`settings_location_sharing`

Reply:
Ah got it! Thank you. Changing that now. @Zelig880

Reply:
Also merged the latest from `main`